back/src/Guide/Types/Edit.hs

```diff
@@ -205,36 +213,102 @@ instance Migrate Edit where
+  type MigrateFrom Edit_v8 = Edit_v7
+  migrate = $(migrateVer ''Edit_v8 7 [
+    CustomM "Edit'AddCategory" [|\x ->
+        Edit'AddCategory_v8
+          { editCategoryUid_v8 = editCategoryUid_v7 x
+          , editCategoryTitle_v8 = editCategoryTitle_v7 x
+          , editCategoryGroup_v8 = toText "Miscellaneous"
+          } |],
+    CopyM 'Edit'AddItem_v8,
+    CopyM 'Edit'AddPro_v8,
+    CopyM 'Edit'AddCon_v8,
+    -- Change category properties
+    CopyM 'Edit'SetCategoryTitle_v8,
+    CopyM 'Edit'SetCategoryGroup_v8,
+    CopyM 'Edit'SetCategoryKind_v8,
+    CopyM 'Edit'SetCategoryNotes_v8,
+    CopyM 'Edit'SetCategoryStatus_v8,
+    CopyM 'Edit'ChangeCategoryEnabledSections_v8,
+    -- Change item properties
+    CopyM 'Edit'SetItemName_v8,
+    CopyM 'Edit'SetItemLink_v8,
+    CopyM 'Edit'SetItemGroup_v8,
+    CopyM 'Edit'SetItemKind_v8,
+    CopyM 'Edit'SetItemDescription_v8,
+    CopyM 'Edit'SetItemNotes_v8,
+    CopyM 'Edit'SetItemEcosystem_v8,
+    -- Change trait properties
+    CopyM 'Edit'SetTraitContent_v8,
+    -- Delete
+    CopyM 'Edit'DeleteCategory_v8,
+    CopyM 'Edit'DeleteItem_v8,
+    CopyM 'Edit'DeleteTrait_v8,
+    -- Other
+    CopyM 'Edit'MoveItem_v8,
+    CopyM 'Edit'MoveTrait_v8
+    ])
+
 -- | Determine whether the edit doesn't actually change anything and so isn't
 -- worth recording in the list of pending edits.
 isVacuousEdit :: Edit -> Bool
-isVacuousEdit Edit'SetCategoryTitle{..} =
-  editCategoryTitle == editCategoryNewTitle
-isVacuousEdit Edit'SetCategoryGroup{..} =
-  editCategoryGroup == editCategoryNewGroup
-isVacuousEdit Edit'SetCategoryNotes{..} =
-  editCategoryNotes == editCategoryNewNotes
-isVacuousEdit Edit'SetCategoryStatus{..} =
-  editCategoryStatus == editCategoryNewStatus
+isVacuousEdit Edit'SetCategoryTitle {..} =
+    editCategoryTitle == editCategoryNewTitle
+isVacuousEdit Edit'SetCategoryGroup {..} =
+    editCategoryGroup == editCategoryNewGroup
+isVacuousEdit Edit'SetCategoryNotes {..} =
+    editCategoryNotes == editCategoryNewNotes
+isVacuousEdit Edit'SetCategoryStatus {..} =
+    editCategoryStatus == editCategoryNewStatus
 isVacuousEdit Edit'ChangeCategoryEnabledSections {..} =
-  null editCategoryEnableSections &&
-  null editCategoryDisableSections
-isVacuousEdit Edit'SetItemName{..} =
-  editItemName == editItemNewName
-isVacuousEdit Edit'SetItemLink{..} =
-  editItemLink == editItemNewLink
-isVacuousEdit Edit'SetItemGroup{..} =
-  editItemGroup == editItemNewGroup
-isVacuousEdit Edit'SetItemHackage{..} =
-  editItemHackage == editItemNewHackage
-isVacuousEdit Edit'SetItemDescription{..} =
-  editItemDescription == editItemNewDescription
-isVacuousEdit Edit'SetItemNotes{..} =
-  editItemNotes == editItemNewNotes
-isVacuousEdit Edit'SetItemEcosystem{..} =
-  editItemEcosystem == editItemNewEcosystem
-isVacuousEdit Edit'SetTraitContent{..} =
-  editTraitContent == editTraitNewContent
+    null editCategoryEnableSections && null editCategoryDisableSections
```

Comment:
Same point as before: this function was consistently formatted and now it's inconsistently formatted.